app/src/ui/delete-branch/delete-branch-dialog.tsx

```diff
@@ -53,7 +55,12 @@ export class DeleteBranch extends React.Component<
         <DialogFooter>
           <ButtonGroup destructive={true}>
             <Button type="submit">Cancel</Button>
-            <Button onClick={this.deleteBranch}>Delete</Button>
+            <Button
+              onClick={this.deleteBranch}
+              disabled={this.state.isDeleting}
```

Comment:
:art: could we do something similar to how we show the commit is taking time by also displaying ` Loading` component?

https://github.com/desktop/desktop/blob/0678a8bd012723ed86f2cad0dcc01044985babb8/app/src/ui/changes/commit-message.tsx#L529-L540

Reply:
Added in #5372.